api/python/cellxgene_census/src/cellxgene_census/_experiment.py

```diff
@@ -31,7 +31,7 @@ def _get_experiment(census: soma.Collection, organism: str) -> soma.Experiment:
         ValueError: if unable to find the specified organism.
 
     Lifecycle:
-        Experimental.
+        Maturing.
```

Comment:
the style used in SOMA is lower case, no punctuation

Eg.,
```
Lifecycle: maturing
```

or

```
Lifecycle:
    maturing
```

Reply:
I prefer the first, as it is much easier to search for.  Not sure if it will mess up online docsite

Reply:
@ebezzi do you know how each will render?

Reply:
I can't find any cases of same-line header+content in our docstrings, so conservatively went with the multi-line form for now.

Reply:
We should use the multi-line. Regardless, this renders as a separate paragraph:
<img width="138" alt="image" src="https://github.com/chanzuckerberg/cellxgene-census/assets/7020189/db303bad-9bb2-4706-84f0-6939dd798a80">
so I think the capitalization and dot makes it look better.

Reply:
To add a bit more context, [this](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html) is the parser we're using for our Google-style docstrings - I believe that without the multiline, the lifecycle section will render as a normal paragraph and not with a section title+paragraph, which will IMHO make the generated docstring page less readable. See [an example](https://tiledbsoma.readthedocs.io/en/latest/_autosummary/tiledbsoma.AxisQuery.html) here.

Reply:
Useful context! I'm going with a compromise: multi-line, but lowercase and no punc, to make everyone equally slightly unhappy. 😆 I'lllet @pablo-gar be the arbitor of docstring fashion upon his return and we can change on a future PR.